AcsEmulator/AcsEmulatorAPI/appsettings.json

```diff
@@ -10,5 +10,16 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "Kestrel": {
+    "Endpoints": {
+      "HttpsInlineCertFile": {
+        "Url": "https://localhost",
+        "Certificate": {
+          "Path": "trouter_selfSigned.pfx",
```

Comment:
How do we best go about this?

Reply:
Curious - why didn't the default asp.net dev certificate work for this?

Reply:
Ah, that would be because the dev certificate has CN=localhost, but we need go.trouter.skype.com for this.